partcad/src/partcad/sketch_factory_dxf.py

```
@@ -50,13 +66,39 @@
+                        wrapper_path,
+                        os.path.abspath(self.path),
+                        os.path.abspath(self.project.config_dir),
+                    ],
+                    request_serialized,
                 )
-                shape = workplane.val().wrapped
+                sys.stderr.write(errors)
+
+                response = base64.b64decode(response_serialized)
+                register_ocp_helper()
+                result = pickle.loads(response)
```

Ruff (0.8.2)

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)